src/AltinnCore/Designer/Dockerfile

```diff
@@ -1,6 +1,7 @@
 FROM node:10.13.0 AS generate-designer-js
 COPY /src/AltinnCore/Designer/package.json .
```

Comment:
Is copy of package.json still needed? If yes, copy both package.json and package-lock,json using a single COPY statement (to avoid creating two layers), if not remove COPY of package.json.

Reply:
package.json is still needed for running npm-scripts.